packages/nc-gui/composables/useLTARStore.ts

```
@@ -200,8 +213,8 @@ const [useProvideLTARStore, useLTARStore] = useInjectionState(
         new Set([
           relatedTableDisplayValueProp.value,
           ...relatedTablePrimaryKeyProps.value,
-          attachmentCol.value?.title,
-          ...(fields.value || [])?.map((f) => f.title as string),
+          ...(attachmentCol.value ? [attachmentCol.value?.title] : []),
+          ...(fields.value || [])?.map((f) => f.title?.trim() as string),
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)